src/hazelcore/semantics/UHPat.re

```diff
@@ -167,3 +167,32 @@ let text_operand =
       u_gen,
     );
   };
+
+let rec is_complete = (pn: t): bool => {
```

Comment:
why is this variable "pn" rather than "p"?

Reply:
fixed